ui/raidboss/raidboss_config.ts

```diff
@@ -1430,15 +1430,26 @@ const flattenTimeline = (
   const dir = filename.slice(0, Math.max(0, lastIndex + 1));
 
   const timelineFile = `${dir}${set.timelineFile}`;
-  delete set.timelineFile;
 
   if (!(timelineFile in files)) {
     console.log(`ERROR: '${filename}' specifies non-existent timeline file '${timelineFile}'.`);
     return;
   }
 
   // set.timeline is processed recursively.
-  set.timeline = [set.timeline, files[timelineFile]];
+  return [set.timeline, files[timelineFile]];
+};
+
+const flattenTimelinePlace = (
```

Comment:
Can this function `delete set.timelineFile` then if you've removed it from the other function, since it's overriding the timeline?